### DISCUSSION TITLE
0.16.4

### DISCUSSION THREAD

**Discussion by SchrodingersGat** [inventree/InvenTree#8213] [Announcements]
### Bug Fixes
* [0.16.x] Enhance exception management (#8174) by @github-actions in https://github.com/inventree/InvenTree/pull/8175
* [0.16.x] Remove translation for logged warnings (#8173) by @github-actions in https://github.com/inventree/InvenTree/pull/8176
* Increase timeout for report printing by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/8187
* Ensure that stock item trees are rebuilt correctly after serialization by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/8193
* [0.16.x] Exception handling for BulkDeleteMixin (#8205) by @github-actions in https://github.com/inventree/InvenTree/pull/8206
* [0.16.x] Add exception handling for default template creatoin (#8209) by @github-actions in https://github.com/inventree/InvenTree/pull/8211
### Other Changes
* Update version.py by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/8158
* [0.16.x] Path management improvements (#8210) by @github-actions in https://github.com/inventree/InvenTree/pull/8212

**Full Changelog**: https://github.com/inventree/InvenTree/compare/0.16.3...0.16.4

<hr /><em>This discussion was created from the release <a href='https://github.com/inventree/InvenTree/releases/tag/0.16.4'>0.16.4</a>.</em>

**Comment by StormTrooper** [comment]
When is this going to be available on apt? Still seeing 0.16.3 as the latest.
